README.md

```diff
@@ -0,0 +1,15 @@
+# fec-infrastructure
+
+Terraform configuration and Concourse pipeline for managing custom FEC infrastructure in AWS.
+
+## Setup
+* Concourse pipeline is deployed to a concourse team owned by 18F Infrastructure
+* Terraform configuration is deployed via Concourse to an AWS account owned by Infrastructure
+* FEC staff are responsible for adding resources that incur a cost to Chandika
+
+## Workflow
+* FEC team member sends a pull request to 18F/fec-infrastructure
+* If applicable, FEC team member sends updated Concourse credentials to infrastructure staff
```

Comment:
FEC might want to update the database name / username / password on an RDS instance.